cmd/internal/cli/singularity.go

```diff
@@ -207,6 +208,22 @@ func sylabsToken(cmd *cobra.Command, args []string) {
 	}
 }
 
+// sylabsRemote returns the remote in use or an error
+func sylabsRemote(filepath string) (*scs.EndPoint, error) {
+	file, err := os.OpenFile(filepath, os.O_RDONLY|os.O_CREATE, 0644)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	c, err := scs.ReadFrom(file)
+	if err != nil {
+		return nil, err
```

Comment:
Same comment here, I think in general we should try to add context to error messages whenever possible